src/scaffold/tests.js

```diff
@@ -0,0 +1,185 @@
+const prettier = require('prettier')
+const { strings } = require('gluegun')
+const { ascTypeForEthereum, ethereumFromAsc } = require("../codegen/types")
+
+const VARIABLES_VALUES = {
+  "i32": 123,
+  "BigInt": 234,
+  "Bytes": 1234567890,
+  "Address": "0x0000000000000000000000000000000000000001",
+  "string": "Example string value",
+  "bool": true,
+}
+
+const generateTestsFiles = (contract, events, indexEvents) => {
+  const event = events[0]
```

Comment:
Why is this variable created? 🤔

Reply:
Yeah, you're right it is unneeded. I merged some code from another PR and reworked some things and I guess I missed that. Thanks for pointing it out!

Reply:
This variable was created because the example test is created only for the first event, similarly how in the mappings only the first event has a full handler example. But it can be removed anyways and just pass events[0] to `generateExampleTest()` function